app/controllers/ops_controller/settings/label_tag_mapping.rb

```diff
@@ -12,8 +14,10 @@ module OpsController::Settings::LabelTagMapping
   # when editing an existing mapping.
   MAPPABLE_ENTITIES = {
     nil                   => nil, # map all entities
-    "Vm"                  => "Amazon",
-    "Image"               => "Amazon",
+    # "Vm"                  => "Amazon",
+    # "Image"               => "Amazon",
+    'Vm'                  => [nil, 'Amazon', 'Azure'].freeze,
+    'Image'               => [nil, 'Amazon', 'Azure'].freeze,
```

Comment:
The key of this hash — e.g. 'Image' — is still what's saved as the ContainerLabelTagMapping `labeled_resource_type`, for each of these options?
When the mapping is performed during refresh, mapper won't be able to distinguish whether user wanted it to apply to only amazon images, azure images or both.

=> You do need distinct `labeled_resource_type` values for each option.
And you'll need to extend ContainerLabelTagMapping::Mapper to apply mappings with several values.  Currently it matches mappings with given type, and nil:
https://github.com/ManageIQ/manageiq/blob/84aae96783c3/app/models/container_label_tag_mapping/mapper.rb#L62-L66
you'll need to match 3 values.